CMakeLists.txt

```diff
@@ -150,9 +150,13 @@ endif()
 option(DEFAULT_FEATURE_STATE "Specify the default state for most optional features" True)
 mark_as_advanced(DEFAULT_FEATURE_STATE)
 
+# openssl/crypto
+option(ENABLE_WOLFSSL "Compile netdata using WolfSSL." False)
+cmake_dependent_option(ENABLE_OPENSSL "Compile netdata using OpenSSL." True "NOT ENABLE_WOLFSSL" False)
```

Comment:
This should just be a `set()` command, not an option. If we’re not using WolfSSL, we by definition _must_ be using OpenSSL. If we decide to make it optional in the future that’s fine, but for now it really isn’t.

Reply:
Done, thank you for your suggestions!